src/p2p/net_node.inl

```diff
@@ -73,16 +73,6 @@ namespace nodetool
     const int64_t default_limit_up = 2048;
     const int64_t default_limit_down = 8192;
     const command_line::arg_descriptor<std::string> arg_p2p_bind_ip        = {"p2p-bind-ip", "Interface for p2p network protocol", "0.0.0.0"};
-    const command_line::arg_descriptor<std::string> arg_p2p_bind_port = {
-        "p2p-bind-port"
-      , "Port for p2p network protocol"
-      , std::to_string(config::P2P_DEFAULT_PORT)
-      };
-    const command_line::arg_descriptor<std::string> arg_testnet_p2p_bind_port = {
-        "testnet-p2p-bind-port"
-      , "Port for testnet p2p network protocol"
-      , std::to_string(config::testnet::P2P_DEFAULT_PORT)
-      };
```

Comment:
Moved it to the header file to be able to include it on cryptonote_core.cpp and use the values in the if statement.

Do you recommend me another way to do it?